src/app/components/categories/category/category.component.html

```diff
@@ -0,0 +1,9 @@
+<section id="category-main">
+  <div class="category-header">
+    <h1 [id]="categories[categoryID].name + '-title'">{{categories[categoryID].name}}</h1>
+    <p><strong>V</strong></p>
```

Comment:
?

Reply:
Same.
And the caps will be changed alongside the slug int he upcoming sprint